src/enums/nameRequestTypes.ts

```diff
@@ -3,6 +3,7 @@ export enum NameRequestTypes {
   AMALGAMATION = 'AML',
   NEW = 'NEW',
   CHANGE_OF_NAME = 'CHG',
-  CONVERSION = 'CNV'
+  CONVERSION = 'CNV',
+  CONTINUATION_IN = 'MVE'
```

Comment:
Would it make sense to use the NrRequestActionCodes enum instead of this local one?

https://github.com/bcgov/bcrs-shared-components/blob/ab4c90c534948621f7cdcb7609c650f889f0f923/src/enums/name-request-codes.ts#L82

And, is this something you think should be done in this ticket, or future?

Reply:
I think future ticket would be better. I will create one.